server/game/cards/08-MotC/WinterCourtHosts.js

```diff
@@ -0,0 +1,23 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl.js');
+
+class WinterCourtHosts extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Draw a card.',
```

Comment:
Sorry more pettiness. :/
Please, no full stops at the end of titles and effect messages.

Reply:
its no problem, should be good now